README.md

```diff
@@ -70,6 +70,17 @@ The different firmwares are inside the repository and not in the realese section
 
 [![WIKI](https://user-images.githubusercontent.com/40640735/217364175-6e3af8df-e6b4-4fcd-9515-09ff02639f64.jpg)](https://github.com/ElectronicCats/CatSniffer/wiki)
 
+## Firmware Repository
+All Catsniffer Firmware has been moved to a different repository, to have a better version control, and issue tracking you will find it here:
+
+https://github.com/ElectronicCats/CatSniffer-Firmware
+
+All catsniffer versions are supported in this repository, you will need to check what version you own and select the proper branch to develop or release for just program your board.
+
+## Software Repository
+All Catsniffer Software has been moved to a different repository, to have a better version control, and issue tracking you will find it here:
```

Comment:
All Catsniffer Software has been moved to a different repository for better version control and issue tracking. You will find it here: